### PR TITLE
fix: asset URL in `/webpack-dev-server` route

### PR DESCRIPTION
**Pull request by snitin315**
- [x] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [x] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

Yes, updated snapshots.

### Motivation / Use-Case

Fix asset URL, `href="index.html"` was redirecting to   `/webpack-dev-server/index.html` instead of `/index.html`.

### Breaking Changes
None

### Additional Info
No

### CODE REVIEW

File: lib/Server.js [hunk 1]
```diff
@@ -1211,11 +1211,16 @@ class Server {
           res.write(`<h2>Compilation: ${name}</h2>`);
           res.write("<ul>");
 
-          const publicPath = item.publicPath === "auto" ? "" : item.publicPath;
+          const publicPath = item.publicPath === "auto" ? "/" : item.publicPath;
```
Comment (alexander-akait) [inline lib/Server.js:1214]: "auto" means nothing before, can you provide example how reproduce the problem?

Comment (snitin315) [reply lib/Server.js:1214]: - `cd ./examples/compression`
- `npx webpack serve --open`
- Go to `localhost:8080/webpack-dev-server`
- Click on main.js

Comment (snitin315) [reply lib/Server.js:1214]: I think we can remove this line change, the code below will do the job.

This is more likely to be an issue with how `href` attribute works,

Comment (alexander-akait) [reply lib/Server.js:1214]: Yep, maybe, I want to look at this in near future

**Review by alexander-akait — APPROVED** [review]

**Comment by snitin315** [comment]
There are conflicts, I will recreate this PR.

**Comment by snitin315** [comment]
This doesn't seem to be an issue anymore. Working fine with master.

**Pull request closed without merging**